test/lib/course_planner/courses/offered_courses_test.exs

```diff
@@ -336,4 +406,28 @@ defmodule CoursePlanner.OfferedCoursesTest do
 
     end
   end
+
+  test "bah" do
```

Comment:
I think you don't need to add this test after the change on line 255, specially with this name 😅